### ISSUE TITLE
[Bug]: cannot see the box number

### ISSUE THREAD

**Issue by zhaodice** [continue-revolution/sd-webui-segment-anything#113]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you updated WebUI and this extension to the latest version?

- [X] I have updated WebUI and this extension to the latest version

### Do you understand that you should read the 1st item of https://github.com/continue-revolution/sd-webui-segment-anything#faq if you cannot install GroundingDINO?

- [X] My problem is not about installing GroundingDINO

### Do you understand that you should use the latest ControlNet extension and enable external control if you want SAM extension to control ControlNet?

- [X] I have updated ControlNet extension and enabled "Allow other script to control this extension"

### Do you understand that you should read the 2nd item of https://github.com/continue-revolution/sd-webui-segment-anything#faq if you observe problems like AttributeError bool object has no attribute enabled and TypeError bool object is not subscriptable?

- [X] My problem is not about such issue, otherwise I have tried changing the extension directory name from sd-webui-segment-anything to a1111-sd-webui-segment-anything

### What happened?

![1](https://github.com/continue-revolution/sd-webui-segment-anything/assets/63996691/fe67e46d-d260-424a-be62-7a7aac396ba5)
The text "Image for GroundingDINO" has covered the number tag , I cannot see the box number...

### Steps to reproduce the problem

always happen when box is too large

### What should have happened?

move the number position to the right , or make the "Image for GroundingDINO" into translucent

### Commit where the problem happens

webui: https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/89f9faa63388756314e8a1d96cf86bf5e0663045
extension: https://github.com/continue-revolution/sd-webui-segment-anything/commit/2ae8c13d6e809c1f3f0ab2e9883e4cfd7619395b

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--xformers
```

### Console logs

```Shell
=================================================================================
You are running xformers 0.0.16rc425.
The program is tested to work with xformers 0.0.17.
To reinstall the desired version, run with commandline flag --reinstall-xformers.

Use --skip-version-check commandline argument to disable this check.
=================================================================================
Additional Network extension not installed, Only hijack built-in lora
LoCon Extension hijack built-in lora successfully
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
2023-06-05 03:56:32,350 - ControlNet - INFO - ControlNet v1.1.217
ControlNet preprocessor location: /home/user/git/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads
2023-06-05 03:56:32,399 - ControlNet - INFO - ControlNet v1.1.217
sd-webui-prompt-all-in-one background API service started successfully.
Loading weights [b42b09ff12] from /home/user/git/stable-diffusion-webui/models/Stable-diffusion/Cetus-Mix.safetensors
Creating model from config: /home/user/git/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(1): easynegative
Model loaded in 2.5s (load weights from disk: 0.4s, create model: 0.8s, apply weights to model: 0.3s, apply half(): 0.3s, load VAE: 0.5s, move model to device: 0.2s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 6.8s (import torch: 0.6s, import gradio: 0.5s, import ldm: 1.0s, other imports: 0.4s, load scripts: 0.9s, create ui: 1.1s, gradio launch: 2.1s).
Found GroundingDINO in pip. Verifying if dynamic library build success.
GroundingDINO dynamic library have been successfully built.
Running GroundingDINO Inference
Initializing GroundingDINO GroundingDINO_SwinT_OGC (694MB)
final text_encoder_type: bert-base-uncased
Found GroundingDINO in pip. Verifying if dynamic library build success.
GroundingDINO dynamic library have been successfully built.
Running GroundingDINO Inference
Initializing GroundingDINO GroundingDINO_SwinT_OGC (694MB)
Found GroundingDINO in pip. Verifying if dynamic library build success.
GroundingDINO dynamic library have been successfully built.
Running GroundingDINO Inference
Initializing GroundingDINO GroundingDINO_SwinT_OGC (694MB)
```

**Comment by continue-revolution** [comment]
This label will be disabled shortly afterwards.

**Comment by continue-revolution** [comment]
Done via https://github.com/continue-revolution/sd-webui-segment-anything/commit/41adc2f46734f55682173a09b8fa6b8442035755
